src/common/mem.c

```diff
@@ -586,6 +586,46 @@ umempobj_get_heapusage(struct umem_pool *ph_p, daos_size_t *curr_allocated)
 	return rc;
 }
 
+/** Obtain the usage statistics for the memory bucket. Note that the usage
+ *  statistics for an evictable memory bucket can be approximate value if
+ *  memory bucket is not yet loaded on to the umem cache.
```

Comment:
so the original umempobj_get_heapusage() is to return total used space (including both non-evict-able & evict-able zones), and this one is to return non-evict-able zones used space or individual evict-able zone used space?

Reply:
yes,  that is correct